Assets/MRTK/Core/Providers/Hands/HandBounds.cs

```diff
@@ -124,10 +136,12 @@ public void OnHandJointsUpdated(InputEventData<IDictionary<TrackedHandJoint, Mix
                         continue;
                     }
 
-                    newBounds.Encapsulate(kvp.Value.Position);
+                    newGlobalBounds.Encapsulate(kvp.Value.Position);
+                    newLocalBounds.Encapsulate(Quaternion.Inverse(palmPose.Rotation) * (kvp.Value.Position - palmPose.Position));
```

Comment:
Is there a reason why the OnSourceDetected method uses the proxy's inverse transform while the OnHandJointsUpdated uses the palmPose "inverse transform?"

Reply:
I am not sure if I 100% understand the question; but I'll outline the reasoning:

- If we have a proxy specified and no hand joints are found, we need to calculate bounds based on the renderer of the specified proxy (because, probably, we have no hand and we are using some kind of controller visualization)
- We still need to compute the local AABB of the proxy/controller visualization in local space.
- Thus, when we compute the local AABB, we use the proxy's inverse transform.

We do not use the hand/palmPose's inverse transform when evaluating the proxy's AABB because the use of a proxy implies that we _don't have a hand._ This is specified explicitly as all of the proxy calculation code is wrapped inside a `if (!handContainsJoints)`. 

In `OnHandJointsUpdated`, we know we're actually using a hand joint-enabled input method, which means we are _not_ using the proxy, and instead we are now computing the local AABB with respect to the hand joint itself (no proxy involved.)

Is this reasoning sound?

Reply:
Ah, that makes sense. I forgot about the case about an input source not having joints. Thank you for the explanation!